### DISCUSSION TITLE
GitHub Actions is currently disabled for your account.

### DISCUSSION THREAD

**Discussion by MatinGhanbari** [community/community#148207] [Actions] [answered]
### Select Topic Area

Bug

### Body

Please help me: 
 GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support](https://support.github.com/contact?tags=dotcom-actions) for assistance.

**Comment by AyoubMabrouk** [comment, marked as answer]
Unfortunately, If GitHub Actions is disabled for your account, you'll need to contact GitHub Support to request activation. Here’s how:

1- Go to the [GitHub Support page](https://support.github.com/).
2- Choose "Contact Support" and sign in if required.
3- Select the category related to GitHub Actions and describe your issue.
4- Submit your request and await their response.

**Comment by MatinGhanbari** [comment]
GitHub support don't answer my ticket. What can I do?

**Reply by AyoubMabrouk** [reply]
Don't worry these are the christmas days so that is why everything is blocked XD
I hope they will reply your ticket soon, best of luck :)

**Comment by happyplazy** [comment]
GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance.

**Comment by happyplazy** [comment]
Please help

**Comment by nojrul21** [comment]
Hello GitHub Organization Admins,

I'm nojru out because my GitHub Actions have been disabled for my account. I'm not sure why this happened, but I need Actions enabled to automate my workflows and run CI/CD pipelines for my projects.

Could you please help me re-enable GitHub Actions for my account? I've checked my account settings and couldn't find any reason for the disablement.

If there's any additional information you need from me or if there's a specific reason for the disablement, please let me know.

Thanks in advance for your help in resolving this issue!
![Screenshot 2025-06-03 104640](https://github.com/user-attachments/assets/4a152d9c-94f5-4254-a85e-5a5e367f24cf)

**Comment by nojrul21** [comment]
Account Name: nojrul21 https://github.com/nojrul21

Hello GitHub Support,

My GitHub Actions have been disabled unexpectedly. I have been using them for personal workflow and testing. I am not violating any policy knowingly. Please review and re-enable GitHub Actions on my account.

GitHub Username: @nojrul21

If there is anything I need to fix or clarify, I am ready to cooperate.

Thank you.

**Comment by zhlhlf** [comment]
I think yours has recovered. How to solve it?

**Reply by MatinGhanbari** [reply]
I registered my request in github support and after several tickets, they fixed this issue for me.

**Comment by Fhgsaaf** [comment]
GitHub Actions is currently disabled for your account. Please reach out to [GitHub Support](https://support.github.com/contact?tags=dotcom-actions) for assistance

**Comment by nojrul21** [comment]
Hello GitHub Organization Admins,

I'm nojru out because my GitHub Actions have been disabled for my account. I'm not sure why this happened, but I need Actions enabled to automate my workflows and run CI/CD pipelines for my projects.

Could you please help me re-enable GitHub Actions for my account? I've checked my account settings and couldn't find any reason for the disablement.

If there's any additional information you need from me or if there's a specific reason for the disablement, please let me know.

Thanks in advance for your help in resolving this issue!
![Uploading Screenshot 2025-08-29 050537.png…]()

**Comment by sourabh623249** [comment]
Hello GitHub Support,

My account is showing the message: "GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance."

Please review my account and re-enable GitHub Actions so I can continue running workflows.

GitHub username: sourabh623249

**Comment by GreensVN** [comment]
Hello GitHub Support,

My account is showing the message: "GitHub Actions is currently disabled for your account. Please reach out to GitHub Support for assistance."
Please review and re-enable GitHub Actions on my account.

Github username: GreensVN

**Comment by ndlovusimphiwemartin-maker** [comment]
Hello GitHub Support,

My GitHub Actions were disabled and I believe this was a mistake.
I am learning GitHub Actions and did not intend to violate any policies.
Please review my account and re-enable GitHub Actions if possible.

Thank you.
